packages/legend-graph/src/models/metamodels/pure/packageableElements/store/relational/mapping/BindingTransformer.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { type Hashable, hashArray } from '@finos/legend-shared';
+import { action, computed, makeObservable, observable } from 'mobx';
+import { CORE_HASH_STRUCTURE } from '../../../../../../../MetaModelConst';
+
+export class BindingTransformer implements Hashable {
```

Comment:
Let's move this class to another directory
`/metamodels/pure/packageableElements/store/binding`
Also, we should move `Binding` to `@finos/legend-graph` now.